doc/design/vic-machine/update.md

```diff
@@ -0,0 +1,207 @@
+# VCH Update (P0)
+This document defines the VCH update command line, and the related changes in vic-machine, to make VCH update process more reasonable.
+
+Here is the key points considerred in this Definition
+- Batch Operation
+
+  We'd like to update VCH configurations more than one item at one time to make the command easy to use
+- Reuse all options used in vic-machine create
+
+  In vic-machine create, we already have 57 options to define VCH configurations, though not all of them can be updated, quite amount of them should be able to be changed, and this number will grow up over time, so we definitely do not want to introduce 3X that number of options to update (update means add/modify/remove)
+
+## Options
+First option is to introduce update primitives
+
+### Delta Options
+#### Update Primitives
+
+- add
```

Comment:
sounds good. The implementation part is similar to what I'm thinking, just in this story, we're not focused on that part, so no more description about it.

For the CLI design, this is an update for anything specified.
And also, the idea for sth really need to be cleared, using new options --clear-registry-cas, --clear-tls-cas.
This might introduce many new options over time, but user can only specify required options in one reconfigure command. So should be better.